src/main/webapp/ProfilePage.html

```diff
@@ -1,38 +1,41 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, shrink-to-fit=no"
-    />
-    <link
-      rel="stylesheet"
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
-    />
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-    <script src="user_script.js"></script>
-    <link
-      rel="stylesheet"
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
-    />
-  </head>
-  <body onload="loadProfile()">
-    <div id="navbar"></div>
-    
-    <div class="container">
-      <h1>My Profile Page</h1>
-      <div class="card" style="width: 22rem;">
-        <img id ="avatar" src="" class="card-img-top" alt="profile avatar">
-        <div class="card-body">
-          <h5 class="card-title" id="name"> </h5>
-          <h6 class="card-subtitle mb-2 " id="username"> </h6>
-          <p class="card-text" id="bio"> </p>
-        </div>
+
+<head>
+	<meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" />
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+	<script src="user_script.js"></script>
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" />
+</head>
+
+<body onload="loadProfile(); loadFavItems();">
+	<div id="navbar"></div>
+
+	<div class="container">
+		<h1 style="text-align:center">My Profile Page</h1>
```

Comment:
Bootstrap has a ```text-center``` class that would be better to use instead of defining the style in the HTML page. https://getbootstrap.com/docs/4.0/utilities/text/